sli/lockptrdatum.h

```diff
@@ -106,7 +106,7 @@ class lockPTRDatum : public lockPTR< D >, public TypedDatum< slt >
     TypedDatum<slt>::operator= is called.
     The TypedDatum = is simply return *this.
   */
-  // lockPTRDatum<D, sli>& operator=(const lockPTRDatum<D, sli>&)
+  lockPTRDatum< D, sli >& operator=( const lockPTRDatum< D, sli >& )
```

Comment:
@JanVogelsang The `;` is missing at the end of this line and that thows clang-format off. Given that this definition was commented out before: Is it needed at all? If not, why not delete it including the comment above?

Reply:
You are right. I thought I was the one who commented out that function, should have checked that before I commented it back in.  
But as it was commented out before as well already, I will now remove it completely as you suggested.